src/lib/openapi/index.ts

```diff
@@ -34,14 +52,30 @@ export const createOpenApiSchema = (
             },
             schemas: {
                 constraintSchema,
+                cloneFeatureSchema,
                 createFeatureSchema,
                 createStrategySchema,
                 featureSchema,
                 featuresSchema,
+                featureEnvironmentInfoSchema,
+                emptyResponseSchema,
                 overrideSchema,
                 parametersSchema,
+                patchAddOperationSchema,
+                patchRemoveOperationSchema,
+                patchReplaceOperationSchema,
+                patchMoveOperationSchema,
+                patchOperationSchema,
+                patchCopyOperationSchema,
+                patchFeatureSchema,
+                patchStrategySchema,
```

Comment:
Looks like these have the same content. Maybe we combine them into something like

```
patchSchema
patchesSchema
patchesRequest
```